docs/rust_wasi_markdown_parser/.gitignore

```diff
@@ -0,0 +1,18 @@
+# Generated Build Output
+wasm32-wasi/
+.rustc_info.json
+
+# Generated by Cargo
+# will have compiled files and executables
+debug/
```

Comment:
I'm a bit confused by the need of all these—you really should only need to exclude `target`, which contains all of these.

I also think all the others below aren't really needed for this example, so perhaps just make this `.gitignore` file match what `cargo new` creates by default—i.e. only include `/target`?